src/components/PlayCollection.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Play from './Play'
+
+export default function PlayCollection({ plays }) {
+
+  return (
+  <>
+  <h1>Plays</h1>
+    <StyledPlayCollection>
+      {plays && plays.map((play, index) => (
+        <Play play={play} key={index} />
+      ))}
+    </StyledPlayCollection>
+  </>
+  )
+}
+
+const StyledPlayCollection = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 10px 0;
+  display: flex;
+  flex-direction: column;
+`
```

Comment:
Vielleicht findest du hier noch einen eindeutigeren key als den index (da der sich ja ändern könnte)?

Reply:
Ich hätte wohl nicht index schreiben sollen. Es handelt sich dabei nur um einen temporären. Dass man Indices nicht als Key verwenden soll, ist mir bekannt. Ich bin mit der Lösung noch nicht zufrieden und werde sie in einem neuen Ticket addressieren.